SDK/simUtil.h

```diff
@@ -48,13 +48,12 @@
 #include "simUtil/Replaceables.h"
 #include "simUtil/ResizeViewManipulator.h"
 #include "simUtil/ScreenCoordinateCalculator.h"
-#include "simUtil/Shaders.h"
-#include "simUtil/SilverLiningSettings.h"
+#include "simUtil/SilverLining.h"
 #include "simUtil/StatsHandler.h"
 #include "simUtil/StatsSizeFixer.h"
 #include "simUtil/StatusText.h"
 #include "simUtil/TerrainToggleEffect.h"
-#include "simUtil/TritonSettings.h"
+#include "simUtil/Triton.h"
```

Comment:
Triton.h not found after a CMake generate and install

Reply:
Sorry, I missed a change.  My working source tree for the changes is not the same that was available to push to github, so I was having to manually copy the changes.  I missed the change to `simUtil/CMakeCache.txt`.  Should be pushed now